keras/src/quantizers/quantizers.py

```diff
@@ -64,7 +66,24 @@ def abs_max_quantize(
     value_range=(-127, 127),
     dtype="int8",
     epsilon=backend.epsilon(),
+    to_numpy=False,
```

Comment:
why isn't `to_numpy` the default if the outputs are the same and it performers better?

Reply:
`abs_max_quantize` is used not only in `quantize`, but also in `AbsMaxQuantizer` which is called in `_int8_call`.
We cannot use numpy for that call.